src/ACSetInterface.jl

```diff
@@ -107,6 +107,8 @@ subpart(acs, names::AbstractVector{Symbol}) =
 
 Base.getindex(acs::ACSet, part, name) = subpart(acs, part, name)
 Base.getindex(acs::ACSet, name) = subpart(acs, name)
+Base.view(acs::ACSet, name) = view(acs,name)
```

Comment:
This line isn't being covered, which is good because it would recurse infinitely!